src/components/index.ts

```diff
@@ -45,3 +45,4 @@ export { Search, SearchProps } from './Search/Search';
 export { Password } from './Password/Password';
 export { VisuallyHidden } from './VisuallyHidden/VisuallyHidden';
 export { RadioButtonProps } from './RadioButton/RadioButtonProps';
+export { Offcanvas, OffcanvasProps} from './Offcanvas/Offcanvas';
```

Comment:
Just noticed this, but there's a space missing at the end of the export which should have been added by prettier. Can you check that it's actually working or if we have an issue?

Reply:
It does work in general, because I saw it did format some part of the code following our prettier. 
I'm not sure why it didn't in that case.